setup.py

```diff
@@ -27,6 +27,7 @@
     "pefile==2021.9.3",
     "pyelftools==0.28",
     "dnfile==0.10.0",
+    "dncil==1.0.0",
```

Comment:
😄

Reply:
should we declare both dnfile and dncil or just dncil?

Reply:
`dncil` doesn't depend on `dnfile` (outside of devleopment) so we need both here.